primefaces-showcase/src/main/java/org/primefaces/showcase/view/panel/DashboardView.java

```diff
@@ -58,9 +58,9 @@ public void init() {
 
         // legacy
         legacyModel = new DefaultDashboardModel();
-        legacyModel.addColumn(new DefaultDashboardColumn(Arrays.asList("sports", "finance")));
-        legacyModel.addColumn(new DefaultDashboardColumn(Arrays.asList("lifestyle", "weather")));
-        legacyModel.addColumn(new DefaultDashboardColumn(Arrays.asList("politics")));
+        legacyModel.addWidget(new DefaultDashboardColumn(Arrays.asList("sports", "finance")));
```

Comment:
shouldnt it then also be DefaultDashboardWidget?

Reply:
You are right and I just moved the code into DefaultDashboardWidget and instead of it was the other way around.